static/css/style.css

```diff
@@ -39,8 +39,31 @@ body {
 -------------------------------------------------- */
 /* Not required for template or sticky footer method. */
 
+.login {
+	padding: 10px 0px 5px 45px; 
+}
+.h1 {
+	border: solid;
+	width: 350px;
+	text-align: center;
+	position: relative; 
+	margin:-100px 0 0 -150px;
+	top: 100px;
+	left: 50%;
+	font-family: 'Carter One', cursive;
+}
+.h2 {
```

Comment:
Same as above - drop the dot, use pure heading tags because somebody already made them for us :)